app/src/main/res/values/attrs.xml

```diff
@@ -11,7 +11,9 @@
     </declare-styleable>
 
     <declare-styleable name="Progress">
-        <attr name="text_size" format="dimension"/>
+        <attr name="android:textColor" format="color"/>
```

Comment:
use global attributes for default names else it will create conflict

Reply:
@danishjamal104 To do so, I just need to move the `attr` tag outside the `declare-styleable` and place it at the top, right?

Reply:
@danishjamal104 I just pushed a new commit and fixed conflicts. Let me know if we are fine.
In case there is something missing, please let me know which view can I use as a reference.